moveit_ros/visualization/planning_scene_rviz_plugin/include/moveit/planning_scene_rviz_plugin/planning_scene_display.h

```diff
@@ -195,6 +194,9 @@ protected Q_SLOTS:
   rviz_common::properties::FloatProperty* scene_display_time_property_;
   rviz_common::properties::EnumProperty* octree_render_property_;
   rviz_common::properties::EnumProperty* octree_coloring_property_;
+
+  // rclcpp node
+  const rclcpp::Node::SharedPtr node_ = rclcpp::Node::make_shared("planning_scene_display");
```

Comment:
Doesn't this imply that rclcpp::init() has been called before, and shouldn't this be rather initialized in the constructor?

Reply:
see previous comment about the `context_` member variable